init.el

```diff
@@ -29,7 +32,7 @@
         'silent 'inhibit-cookies)
       (goto-char (point-max))
       (eval-print-last-sexp)))
-  (load bootstrap-file nil 'nomessage))
+  (load bootstrap-file 'noerror 'nomessage))
```

Comment:
This ***must*** return an error, otherwise `straight.el`'s bootstrap file is missing and most of the rest of this configuration cannot work.

While the previous lines are explicitly setting up `straight.el`, it might still fail if the `user-emacs-directory` is read-only or similar. Or `install.el` might contain `nil` due to a broken repository. In either cases, we don't have `straigth.el` available and *really* should know about that.

Please revert this change.